docs/sql-migration-guide-upgrade.md

```diff
@@ -23,6 +23,8 @@ license: |
 {:toc}
 
 ## Upgrading From Spark SQL 2.4 to 3.0
+  - Since Spark 3.0, the string concatenation operator(`||`) has lower precedence than other operators.
```

Comment:
Is it not "has the lowest precedence among operators"? Also, could you add a simple example query in this statement?